swesmith/profiles/rust.py

```diff
@@ -115,6 +114,106 @@ class Uuid2fd9b614(RustProfile):
     test_cmd: str = "cargo test --verbose --all-features"
 
 
+@dataclass
+class MdBook37273ba8(RustProfile):
+    owner: str = "rust-lang"
+    repo: str = "mdBook"
+    commit: str = "37273ba8e0f86771b02f3a8a4bd3b0b3d388c573"
+    test_cmd: str = "cargo test --workspace --verbose"
+
+
+@dataclass
+class RustCSVda000888(RustProfile):
+    owner: str = "BurntSushi"
+    repo: str = "rust-csv"
+    commit: str = "da0008884062cf222ceb9c05f006be4bb6ac38a7"
+
+
+@dataclass
+class Html5everb93afc94(RustProfile):
+    owner: str = "servo"
+    repo: str = "html5ever"
+    commit: str = "b93afc9484cf5de40b422a44f9cea86ab371e3ee"
+
+    @property
+    def dockerfile(self):
+        return f"""FROM rust:1.88
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=Etc/UTC
+
+RUN apt update && apt install -y wget git build-essential \
+&& rm -rf /var/lib/apt/lists/*
+
+RUN git clone https://github.com/{self.mirror_name} /testbed
+WORKDIR /testbed
+RUN git submodule update --init
+"""
+
+
+@dataclass
+class Byteorder5a82625f(RustProfile):
+    owner: str = "BurntSushi"
+    repo: str = "byteorder"
+    commit: str = "5a82625fae462e8ba64cec8146b24a372b4d75c6"
+
+
+@dataclass
+class Chronod43108cb(RustProfile):
+    owner: str = "chronotope"
+    repo: str = "chrono"
+    commit: str = "d43108cbfc884b0864d1cf2db7719aedf4adbf23"
+
+
+@dataclass
+class Rpds3e7c8ae6(RustProfile):
+    owner: str = "orium"
+    repo: str = "rpds"
+    commit: str = "3e7c8ae693cdc6e1b255c87279b6ad8aded6401d"
+
+
+@dataclass
+class Itertools041c733c(RustProfile):
+    owner: str = "rust-itertools"
+    repo: str = "itertools"
+    commit: str = "041c733cb6fbfe6aae5cce28766dc6020043a7f9"
+
+
+@dataclass
+class Rayon1fd20485(RustProfile):
+    owner: str = "rayon-rs"
+    repo: str = "rayon"
+    commit: str = "1fd20485bd0bb55541d8080a31e104c7b758cb48"
+
+
+@dataclass
+class Ripgrep3b7fd442(RustProfile):
```

Comment:
As discussed in slack perhaps we don't add `burntsushi/ripgrep` as it is in SWE-bench_Multilingual.

Reply:
Added `eval_sets` as well here :)